tutorials/developer-tutorials/createNewIdentityAttributes.md

```diff
@@ -0,0 +1,72 @@
+
+In the very first tutorial (Tutorial 1: Local smart contract development), we used a pre-set identity (`admin`) to connect to the Local Fabric gateway and submit/evaluate transactions. In this tutorial, we will learn how to create _our own_ identity with attributes.
+
+Having multiple identities enrolled with the Local Fabric CA (Certificate Authority) allows you to represent different participants in a blockchain network. For example two identities `Tesla` (the manufacturer) and `Jack` (the customer) may be permitted to submit different transactions in a smart contract.
+
+Created identities can be assigned attributes, which allows attribute-based access control. A transaction function could be written such that if the value of a given attribute on the identity being used matches a particular value, it will do one thing; if the value does not match, it could do something else, or throw an error. How to write such logic into smart contract transactions will be covered in a later tutorial.
+
+As an example, suppose we have a smart contract which allows participants to record the cars owned by a company on a blockchain. We may allow participants to add cars to the blockchain by submitting a `createCar()` transaction. Now, this would be fine for `Tesla` because it is a manufacturer, but since `Jack` is an individual, if he tries to submit a `createCar()` transaction, we should stop him! Here’s how you would register and enrol an identity for `Tesla` from the Local Fabric CA with suitable attributes…
+
+1. In the `FABRIC ENVIRONMENTS` panel, look for `ca.org1.example.com` (it's under Nodes), right click it and choose `Create Identity (register and enroll)`. 
+
+2. You will be asked to provide a name for your identity. For the purpose of this tutorial, we will call our identity `Tesla` and select `Yes` when asked to add attributes. We will give Tesla the attributes `[{"name": "manufacturer", "value": "true", "ecert":true}]`
+
+3. Upon submitting your request, you should see a dialog box at the bottom right of the screen confirming that your identity `Tesla` has been created with attributes `[{"name": "manufacturer", "value": "true", "ecert":true}]`. The newly created identity should also appear in the `Fabric Wallets` panel under `Local Fabric Wallet`. 
+
+4. To connect to the Local Fabric gateway using this identity, you simply click `Local Fabric` under the Fabric Gateways panel and select the identity you wish to connect with (Tesla in our case).
+> Command Palette alternative: `Create Identity (register and enroll)`
+</details>
+
+<details>
+<summary><b>2. Using our new identity</b></summary>
+
+Now that we've created our very own identities, we can implement attribute-based action control. To do so, follow the steps below:
+
+1. Click the `Explorer` icon in the left sidebar (it will probably be at the top and will look like a folder) and navigate to the `car-contract.ts` file in the `src` folder under the contract directory.
```

Comment:
Where does it say to create a smart contract with a car asset?

Reply:
Edited - Added steps on how to create a smart contract with a car asset.